checks/raw/fuzzing.go

```diff
@@ -15,14 +15,48 @@
 package raw
 
 import (
+	"bytes"
 	"fmt"
+	"regexp"
+	"strings"
 
 	"github.com/ossf/scorecard/v4/checker"
 	"github.com/ossf/scorecard/v4/checks/fileparser"
 	"github.com/ossf/scorecard/v4/clients"
 	sce "github.com/ossf/scorecard/v4/errors"
 )
 
+const (
+	fuzzNameOSSFuzz         = "OSSFuzz"
+	fuzzNameClusterFuzzLite = "ClusterFuzzLite"
+	fuzzNameBuiltInGo       = "GoBuiltInFuzzer"
+	// TODO: add more fuzzing check supports.
+)
+
+type filesWithPatternStr struct {
+	pattern string
+	files   []checker.File
+}
+type languageFuzzConfig struct {
+	langFuzzDocumentURL, langFuzzDesc                        *string
```

Comment:
shorten these strings. `languageFuzzConfig` is expressive enough. The fields can be `URL`, `Desc`, `Name`, etc

Reply:
resolved